src/components/message.tsx

```diff
@@ -7,8 +7,7 @@
 import * as React from 'react';
 import withIntl from './injectIntl';
 import {MessageDescriptor, IntlShape} from '../types';
-import * as shallowEquals_ from 'shallow-equal/objects';
-const shallowEquals = shallowEquals_;
+const shallowEquals = require('shallow-equal/objects');
```

Comment:
Why require?

Reply:
bc those deps don't have typings & rollup complains if you `import * as foo` and then use `foo()` (namespace isn't technically a function)